docs/run-nym-nodes/nodes/gateways.md

```diff
@@ -63,11 +63,11 @@ You can check that your binaries are properly compiled with:
 <details>
   <summary>console output</summary>
 
-        nym-gateway 1.0.1
+        nym-gateway 1.1.0
         Nymtech
 
         Build Timestamp:    2022-05-06T13:07:46.187796508+00:00
-        Build Version:      1.0.1
+        Build Version:      1.1.0
         Commit SHA:         945dda0c24f2f964f27066af320441446973e383
```

Comment:
are these console outputs correct? 
* You'll need to change the Commit SHA to match those actually printed in the console on running these commands (this comment goes for all instances of this command output) 
* likewise with the `build timestamp`